security-jwks/src/main/java/io/airlift/security/jwks/JwksDecoder.java

```diff
@@ -71,14 +71,13 @@ public static Optional<JwkRsaPublicKey> tryDecodeRsaKey(String keyId, JsonKey ke
         // alg field is optional so not verified
         // use field is optional so not verified
         Optional<BigInteger> modulus = key.getStringProperty("n").flatMap(encodedModulus -> decodeBigint(keyId, "modulus", encodedModulus));
-        if (!modulus.isPresent()) {
+        if (modulus.isEmpty()) {
             return Optional.empty();
         }
         Optional<BigInteger> exponent = key.getStringProperty("e").flatMap(encodedExponent -> decodeBigint(keyId, "exponent", encodedExponent));
```

Comment:
Inline this so that we can use the good name for the lambda argument
```java
return key.getStringProperty("e")
        .flatMap(exponent -> decodeBigint(keyId, "exponent", exponent))
        .map(exponent -> new JwkRsaPublicKey(keyId, exponent, modulus.get()));